docs/standards/hcs-2.md

```diff
@@ -1,14 +1,10 @@
----
-description: This specification provides a standard way to create a new registry for a sub-standard like HCS-1.
----
-
-# HCS-2 Standard: Topic Registries
+# HCS-2 Standard: Advanced Topic Registries
```

Comment:
Also wondering, if we should have a "primary" author "coauthor" on these standards. In this case, I think you would be primary since you wrote it